firenoc/firenoc-services/src/utils/index.js

```
@@ -204,16 +222,16 @@ let SCHEMA_REPLACE_STRING = "{schema}";
  * 
  * else it's tenant level
  */
-let stateLevelTenantIdLength = 1;
+let stateLevelTenantIdLength = parseInt(envVariables.STATE_LEVEL_TENANTID_LENGTH);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'stateLevelTenantIdLength' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)